docs/supervisor_and_streams.rst

```diff
@@ -60,12 +70,21 @@ Change the directory group ownership to ``www-data``:
 
         $ sudo chown www-data:www-data /var/log/sfm
 
-Edit local_settings.py to set DATA_DIR to the directory where you
-want stream output stored. Set SUPERVISOR\_PROCESS\_OWNER to a user
+Edit local_settings.py to set SUPERVISOR\_PROCESS\_OWNER to a user
 who has rights to write to ``/var/log/sfm`` (such as your user).
```

Comment:
@kerchner We can mention this to be www-data here(ie. instead of such as your user we can say such as your user: www-data) as that was what we used in above steps if i am right here.